### PR TITLE
TMode 

### PR DESCRIPTION
**Pull request by Trmotton**
I'm still trying to get the hang Off it.... Maybe I'll get a better understanding as I go through this website Ginhub....?

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+> Please provide a description of the bug / issue, and provide the details below:
+
+====================================================================
+
+###### If the issue is visual, please provide screenshots here
```
Comment (Trmotton) [inline ISSUE_TEMPLATE.md]: I'm trying to get end idea of what the hell am I doing?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+====================================================================
+
+###### Steps to reproduce the problem
```
Comment (Trmotton) [inline ISSUE_TEMPLATE.md]: Next week I might take my girl out to the movies,... Oi wait hold on I have no chick!

**Comment by kenwheeler** [comment]
wat

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Trmotton) [inline ISSUE_TEMPLATE.md]: All this is saying leave a line of notes across

**Pull request closed without merging**